content/en/case-studies/gw-discov.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Case Study: Discovery of Gravitational Waves"
+sidebar: false
+---
+
+{{< figure src="/images/content_images/cs/gw_sxs_image.png" caption="**Gravitational Waves**" alt="gwimg" attr="(**Image Credits: The Simulating eXtreme Spacetimes (SXS) Project at LIGO** )" attrlink="https://youtu.be/Zt8Z_uzG71o" >}}
```

Comment:
Same question about `alt` text, can you make it more descriptive?

Reply:
done